config/betterquesting/DefaultQuests/Quests/BiofortheMasses-AAAAAAAAAAAAAAAAAAAAFQ==/PyrolyseOven-AAAAAAAAAAAAAAAAAAAMIA==.json

```diff
@@ -99,11 +99,7 @@
           "Count:3": 60,
           "Damage:2": 2,
           "OreDict:8": "",
-          "id:8": "betterquesting:placeholder",
-          "tag:10": {
-            "orig_id:8": "dreamcraft:gt.blockcasingsNH",
-            "orig_meta:3": 2
-          }
+          "id:8": "gregtech:gt.blockcasingsNH"
```

Comment:
I'm not sure what happened here but I think that a PR shuffling quest lines around accidentally had some of the casing PRs in it causing these quests to get placeholder'd. If someone wants to fix this temporarily for master they can, I will handle the merge conflict here again. Otherwise this should fix the issue (though I haven't tested as I'm unable to since I am out of town)

Reply:
just one of the problems with pre-heavy nightlies. normally its easy to catch, but not in a +200k, -200k PR XD